course_info/templates/course_info/partials/people.html

```diff
@@ -146,7 +146,8 @@
 								</a>
 							</div>
 
-              <div class="col-md-8 col-md-offset-2 hidden" style="padding-top: 1em">
+              <div class="col-md-8 col-md-offset-2"
+                   style="padding-top: 1em" ng-class="{'hidden' : !searchInProgress || searchResults.length > 0}">
```

Comment:
Ahhh, it's a map of classes to expressions that determine whether that class is active or not.  Clever.